lsp-mode.el

```diff
@@ -3072,7 +3072,7 @@ This method is used if we do not have `buffer-replace-content'."
                              (prepare-change-group)))
              (howmany (length edits))
              (reporter (make-progress-reporter
-                        (lsp--info "Applying %s edits to `%s'..."
+                        (lsp--info "Applying %s edits to %s ..."
```

Comment:
Do you use the smartparens?

My environment is macos catalina, and emacs 26.3. 
I will keep looking if that appear again.

Reply:
The error happened again, so I just handle the case with the default message while progress report.

Reply:
Instead of depending on the `lsp--info` result, can you introduce new variable and asign it to the result of applying `(format "Applying ..."` and then use this variable for both the progress reported and the info call? This will make it a bit clearer.